integration/tests/upgrades/version_beacon_service_event_test.go

```diff
@@ -0,0 +1,194 @@
+package upgrades
+
+import (
+	"context"
+	"testing"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-core-contracts/lib/go/templates"
+
+	sdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go/model/flow"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type TestServiceEventVersionControl struct {
+	Suite
+}
+
+func (s *TestServiceEventVersionControl) TestEmittingVersionBeaconServiceEvent() {
+
+	// n version 0.3.7
```

Comment:
do we need to update this test on each release?  if updated is needed each time, add a comment, and maybe define the constants outside the test.

Reply:
no, this is just some random version. the test is completely self contained.